packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -32,6 +32,7 @@ import { updateMessagesWithAttached } from './utils/updateMessagesWithAttached';
 import { FileMetadata } from '@internal/react-components';
 /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
 import { ChatAttachment, ChatAttachmentType } from '@azure/communication-chat';
+import { ChatParticipant } from '@azure/communication-chat';
```

Comment:
Could you update `import` to `import type` as we probably can import type only instead of the whole object here?

Reply:
Done